kanopi/wp-vanilla-theme
# Sm uglify js

## Description
> As a developer, I need JS to not only be uglified, but also transpiled so it’s compatible with older browsers.

Currently the theme does not transpile JS.

## Affected URL
n/a

## Related Tickets
n/a

## Steps to Validate
1. Add JS to theme which is only supported in relatively newer browsers, such as [Element.matches()](https://developer.mozilla.org/en-US/docs/Web/API/Element/matches).
2. Run `gulp js`.
3. Verify there are no compilation errors and a file is written to `/dist/js/scripts.js`
4. Test the code in a browser to verify it works.
